util/test/chplExperimentGatherUtils/extractCompileAndExecTimeFromStartTest.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python
```

Comment:
you know my wish already, but it'd be really great if this could read the performance keys that are reported by the test itself, rather than looking at what start_test results. Which are only relevant for shootouts as far as I know.